lib/miq_tools_services/bugzilla.rb

```diff
@@ -26,7 +40,7 @@ def service
     def self.ids_in_git_commit_message(message)
       ids = []
       message.each_line.collect do |line|
-        match = %r{^(Fixes|Resolves)?\s*https://bugzilla\.redhat\.com//?show_bug\.cgi\?id=(?<bug_id>\d+)$}.match(line)
+        match = /^((#{CLOSING_KEYWORDS.join("|")}):?)?\s*#{URL_REGEX}$/i.match(line)
```

Comment:
should we extract the creation of the RegEx into a constant on the same level as `URL_REGEX`?

`MATCH_REGEX = /^((#{CLOSING_KEYWORDS.join("|")}):?)?\s*#{URL_REGEX}$/i`

then this line becomes:

` match = MATCH_REGEX.match(line)`